builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -591,74 +590,73 @@ def checkoutSCM(repo_name) {
  * @param  serviceName
  * @return
  */
-def updateServiceNameConfig() {
```

Comment:
`updateServiceNameConfig` also has been combined into `writeServerlessFile` - is it not called elsewhere? Also, is `writeServerlessFile` called else where which could impact it?

Reply:
No. As discussed it is good to go